packages/smithy-core/src/smithy_core/traits.py

```diff
@@ -193,3 +193,117 @@ def __post_init__(self):
     @property
     def value(self) -> str:
         return self.document_value  # type: ignore
+
+
+# TODO: Get all this moved over to the http package
```

Comment:
_Does_ it need to be? I don't think we need to put traits in their "respective" packages

Reply:
I'd tend to agree with this, I think the other PR we moved everything into the centralized location. I like that unless we have a technical limitation of keeping them together. Is the concern that we may include traits that aren't needed if the smithy-http package isn't present, or are we hitting typing problems?

Reply:
It's more about fighting against package size inflation, though practically for HTTP it's not much of an issue.